crates/bevy_core_pipeline/src/bloom/settings.rs

```diff
@@ -0,0 +1,168 @@
+use bevy_ecs::prelude::Component;
+use bevy_reflect::Reflect;
+
+/// Applies a bloom effect to a 2d or 3d camera.
+///
+/// Bloom emulates an effect found in real cameras and the human eye,
+/// causing halos to appear around very bright parts of the scene.
+///
+/// Often used in conjunction with `bevy_pbr::StandardMaterial::emissive`.
+///
+/// Bloom is best used alongside a tonemapping function that desaturates bright colors,
+/// such as ACES Filmic (Bevy's default).
+///
+/// See also <https://en.wikipedia.org/wiki/Bloom_(shader_effect)>.
+/// 
+/// Bevy's implementation uses a parametric curve to blend between a set of
+/// blurred (lower freequency) images generated from input image.
+/// See <https://starlederer.github.io/bloom/> for a vizualization of the parametric curve
+/// used in Bevy as well as a vislualization of the curve's respective scattering profile.
+#[derive(Component, Reflect, Clone)]
+pub struct BloomSettings {
+    /// Controls the baseline of how much the image is scattered (default: 0.3).
+    /// 
+    /// # In energy-conserving mode
+    /// The value represents how likely the light is to scatter.
+    /// 
+    /// The value should be clamed between 0.0 and 1.0 where:
```

Comment:
Btw, for some of the other settings, iirc we clamp in the code before passing it to the shader. Should we do that for these too? If we don't already. I'll leave it to you, as you're more familiar with each parameter, but just make sure we clamp where needed.

Reply:
> iirc we clamp in the code before passing

I don't think that was the case with the original bloom implementation. I remember playing with negative intensity.

The values do have valid ranges but some only in energy-conserving mode. I think clamping should be done by user/ui because doing that at runtime is a bit of a resource waste